src/Terminal/UnixTerminal.php

```diff
@@ -163,6 +163,10 @@ public function getKeyedInput()
             "\n"     => 'enter',
             "\r"     => 'enter',
             " "      => 'enter',
+            "\033[D" => 'left',
```

Comment:
I am not very sure,because the need for left and right arrow keys
The result is that I refer to the previous key-value debugging print out